src/group_impl.h

```diff
@@ -672,17 +727,17 @@ static void secp256k1_gej_add_ge(secp256k1_gej *r, const secp256k1_gej *a, const
      */
 
     secp256k1_fe_sqr(&zz, &a->z);                       /* z = Z1^2 */
-    u1 = a->x; secp256k1_fe_normalize_weak(&u1);        /* u1 = U1 = X1*Z2^2 (1) */
+    u1 = a->x;                                          /* u1 = U1 = X1*Z2^2 (6) */
```

Comment:
This sounds like a good idea, as it would avoid the tedious work of adapting the comments if the magnitude limits ever change in the future. On the other hand, the expressions get quite long at the end, e.g. `((GEJ_Y_M+3)/2 + 1)` for the last annotation in `secp256k1_gej_add_ge` (instruction `secp256k1_fe_half(&r->y);`), not sure if this is still that helpful for the reader in constrast to a concrete magnitude number.

Reply:
I think the version with constants is slightly better.